pkgs/by-name/in/insync-nautilus/package.nix

```
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  dpkg,
+  gnome,
+  insync
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Makes this overrideable